scripts/zones/Balgas_Dais/IDs.lua

```diff
@@ -18,28 +18,23 @@ zones[xi.zone.BALGAS_DAIS] =
         LOGIN_CAMPAIGN_UNDERWAY       = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
-        CONQUEST_BASE                 = 7060, -- Tallying conquest results...
-        PARTY_MEMBERS_HAVE_FALLEN     = 7569, -- All party members have fallen in battle. Now leaving the battlefield.
-        THE_PARTY_WILL_BE_REMOVED     = 7576, -- If all party members' HP are still zero after # minute[/s], the party will be removed from the battlefield.
-        YOU_DECIDED_TO_SHOW_UP        = 7642, -- So, you decided to show up. Now it's time to see what you're really made of, heh heh heh.
-        LOOKS_LIKE_YOU_WERENT_READY   = 7643, -- Looks like you weren't ready for me, were you? Now go home, wash your face, and come back when you think you've got what it takes.
-        YOUVE_COME_A_LONG_WAY         = 7644, -- Hm. That was a mighty fine display of skill there, <name>. You've come a long way...
-        TEACH_YOU_TO_RESPECT_ELDERS   = 7645, -- I'll teach you to respect your elders!
-        TAKE_THAT_YOU_WHIPPERSNAPPER  = 7646, -- Take that, you whippersnapper!
-        NOW_THAT_IM_WARMED_UP         = 7647, -- Now that I'm warmed up...
-        THAT_LL_HURT_IN_THE_MORNING   = 7648, -- Ungh... That'll hurt in the morning...
-        MACAN_WHISKERS_SLIGHTLY       = 7666, -- Gadangan's whiskers begin to twitch slightly.
-        MACAN_WHISKERS_RAPIDLY        = 7667, -- Gadangan's whiskers begin to twitch rapidly.
-        MACAN_WHISKERS_WILDLY         = 7668, -- Gadangan's whiskers begin to twitch wildly.
-        MACAN_WHISKERS_VIOLENTLY      = 7669, -- Gadangan's whiskers begin to twitch violently.
-        MACAN_WHISKERS_UNCONTROLLABLY = 7670, -- Gadangan's whiskers begin to twitch uncontrollably.
-        PROMISE_ME_YOU_WONT_GO_DOWN   = 7686, -- Promise you won't go down too easy, okay?
-        IM_JUST_GETTING_WARMED_UP     = 7687, -- Haha! I'm just getting warmed up!
-        YOU_PACKED_MORE_OF_A_PUNCH    = 7688, -- Hah! You pack more of a punch than I thoughtaru. But I won't go down as easy as old Maat!
-        WHATS_THIS_STRANGE_FEELING    = 7689, -- What's this strange feeling...? It's not supposed to end...like...
-        HUH_IS_THAT_ALL               = 7690, -- Huh? Is that all? I haven't even broken a sweataru...
-        YIKEY_WIKEYS                  = 7691, -- Yikey-wikeys! Get that thing away from meee!
-        WHATS_THE_MATTARU             = 7692, -- <Pant, wheeze>... What's the mattaru, <name>? Too much of a pansy-wansy to fight fair?
```

Comment:
A lot of these were removed during merge conflicts which are needed inside `Macan_Gadangan.lua`

The new ID should be verified with POLUtils or the update extractor.